src/incremental/compareCIL.ml

```diff
@@ -94,16 +95,20 @@ let compareCilFiles ?(eq=eq_glob) (oldAST: file) (newAST: file) =
       let ident = identifier_of_global global in
       let old_global = GlobalMap.find ident map in
       (* Do a (recursive) equal comparison ignoring location information *)
-      let change_status, unchangedHeader, diff = eq old_global global cfgs in
-      let append_to_changed () =
+      let change_status, diff = eq old_global global cfgs in
+      let append_to_changed (unchangedHeader) =
         changes.changed <- {current = global; old = old_global; unchangedHeader; diff} :: changes.changed
       in
       match change_status with
-      | Changed -> append_to_changed ()
+      | ChangedFunHeader f ->
+        changes.exclude_from_rel_destab <- VarinfoSet.add f.svar changes.exclude_from_rel_destab;
+        append_to_changed false
```

Comment:
`append_to_changed false` looks weird, as if the `false` argument indicates not appending. Would be clearer with a named argument: `append_to_changed ~unchangedHeader:false`.